components/header/components/DotcmsLogo.tsx

```diff
@@ -2,38 +2,40 @@ import React from 'react';
-            height="23"
-            viewBox="0 0 129 23"
-            width="129"
-            xmlns="http://www.w3.org/2000/svg"
-        >
-            <path
-                d="M8.27677 23H8.36263C11.7111 22.9828 13.8747 21.009 15.3515 18.7776V22.6052H17V0H15.3515V9.16567C13.9434 7.00299 11.7283 5.06343 8.31111 5.06343C4.20707 5.06343 0.0515152 8.41045 0 13.9373V14.2119C0.0515152 19.7045 4.1899 22.9828 8.27677 23ZM1.76869 13.9545C1.76869 9.35448 4.75657 6.64254 8.48283 6.64254C12.0374 6.64254 15.4545 9.59478 15.4545 13.9888V14.0575C15.4545 18.4172 12.0374 21.4037 8.48283 21.4037C4.89394 21.4037 1.76869 18.5201 1.76869 14.0231V13.9545Z"
-                fill="#0A0724"
-            />
-            <path
-                d="M41.2081 18.2934V7.02535H39V5.4966H41.2081V0H42.6701V5.4966H48V7.00817H42.6701V18.1044C42.6701 20.5607 43.9492 21.4539 45.7005 21.4539C46.4467 21.4539 47.0711 21.2822 47.9391 20.8527V22.3987C47.1624 22.8109 46.4162 22.9998 45.4721 22.9998C43.1117 23.017 41.2081 21.5742 41.2081 18.2934Z"
-                fill="#0A0724"
-            />
-            <path
-                d="M52 11.5945V11.5086C52 5.20462 56.6949 0 63.2881 0C67.339 0 69.8814 1.40851 71.8814 3.71022L68.1525 7.8327C66.7797 6.35548 65.4237 5.39357 63.2542 5.39357C60.2034 5.39357 58.0339 8.15906 58.0339 11.4227V11.5086C58.0339 14.8924 60.1695 17.6064 63.4915 17.6064C65.5424 17.6064 66.9492 16.696 68.4237 15.2531L72 18.9634C69.9153 21.3166 67.5085 23 63.2034 23C56.7288 23.0171 52 17.8984 52 11.5945Z"
-                fill="#0A0724"
-            />
-            <path
-                d="M75 0.406154H80.9405V3.46923C82.319 1.65846 84.1077 0 87.1272 0C89.8677 0 91.9354 1.25231 93.0349 3.41846C94.8728 1.20154 97.0718 0 99.9272 0C104.342 0 107 2.74154 107 7.93692V22H101.059V9.95077C101.059 7.05692 99.8123 5.55077 97.5805 5.55077C95.3487 5.55077 93.9867 7.04 93.9867 9.95077V22H88.0462V9.95077C88.0462 7.05692 86.799 5.55077 84.5672 5.55077C82.3354 5.55077 80.9733 7.04 80.9733 9.95077V22H75.0328V0.406154"
-                fill="#0A0724"
-            />
-            <path
-                d="M112 18.7667L114.542 14.8C116.809 16.4667 119.189 17.35 121.148 17.35C122.864 17.35 123.641 16.7167 123.641 15.7667C123.641 14.3833 121.617 13.9333 119.302 13.2333C116.371 12.3667 113.052 10.9667 113.052 6.83333C113.052 2.41667 116.485 0 120.71 0C123.366 0 126.264 0.916667 128.53 2.46667L126.264 6.63333C124.191 5.4 122.119 4.65 120.597 4.65C119.156 4.65 118.411 5.28333 118.411 6.11667C118.411 7.38333 120.403 7.95 122.67 8.73333C125.6 9.73333 129 11.15 129 15.05C129 19.85 125.519 22 120.986 22C118.055 21.9833 114.769 20.9833 112 18.7667Z"
-                fill="#0A0724"
-            />
-            <path
-                d="M28.5333 5C23.6 5 20 9.18286 20 14V14.0686C20 18.8857 23.5667 23 28.4666 23C33.4 23 36.9999 18.8171 36.9999 14V13.9314C37.0166 9.11429 33.4333 5 28.5333 5Z"
-                fill="#C336E5"
-            />
-        </svg>
+        <div className="w-72 lg:pl-6">
```

Comment:
Adding this in the logo component make it less reusable. This should live outside the logo.

Also are you sure you need the `w-72`?